InnoClinic.OfficesAPI/BLL/Helpers/OfficeHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using BLL.DTO;
+using BLL.Exceptions;
+using DAL.Entities;
+using DAL.Interfaces;
+using Microsoft.AspNetCore.Http;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace BLL.Helpers;
+
+public class OfficeHelper(IOfficeRepository officeRepository, IPhotoRepository photoRepository, IMapper mapper)
```

Comment:
Helpers should be static, if you can't make it static because of some methods - place these methods in related services, but make them private